index.html

```diff
@@ -1,12 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+
 <body>
-    
+    <!--banner powinien  być pomiędzy nav a header-->
+    <div class="banner"><img src="https://i.im.ge/2022/07/28/FmFMH9.jpg" /></div>
```

Comment:
A czy baner nie powinien być właśnie w header?

Reply:
W sumie tak, Kondrad mógłby dodać to wtedy w header  :)? w sumie jeśli zaakceptujecie i spullujecie na lokalny to nie powinno być konfliktu jeśli osadzicie to następnie w header? co sądzicie?

Reply:
Złą praktyką jest mergowanie czegoś do poprawy :). Jeśli nie możesz tego zmienić, to ktoś może przełączyć się na Twojego branche i zmiany wprowadzić :)

Reply:
ok wszystko jasne poprawie! :)